challenge.js

```diff
@@ -9,7 +9,7 @@
  */
 
 // ANSWER
-
+ use load() function to load the contents of another HTML file on the server and place it anywhere on the current page so you can replace the current HTML if needed.
```

Comment:
Are you referring to the load event? We did not discuss any load function. We very much want to avoid replacing HTML; our goal is to alter the DOM without having to request a new page from the server.